Markup/samples/PickUp/css/pickup.css

```diff
@@ -0,0 +1,102 @@
+/*
+~   Copyright (c) 2018 WSO2 Inc. (http://wso2.com) All Rights Reserved.
+~
+~   Licensed under the Apache License, Version 2.0 (the "License");
+~   you may not use this file except in compliance with the License.
+~   You may obtain a copy of the License at
+~
+~        http://www.apache.org/licenses/LICENSE-2.0
+~
+~   Unless required by applicable law or agreed to in writing, software
+~   distributed under the License is distributed on an "AS IS" BASIS,
+~   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~   See the License for the specific language governing permissions and
+~   limitations under the License.
+*/
+
+/* Login page */
+
+.btn-primary {
+    color: #fc511f !important;
+}
+
+.pickup-btn {
+    width: 100%;
+    background: #fc511f !important;
```

Comment:
Do we really want to use important?

Reply:
Need to add important as the button primary styles overides it.